app/jobs/handover_follow_up_job.rb

```diff
@@ -0,0 +1,32 @@
+class HandoverFollowUpJob < ApplicationJob
+  queue_as :default
+
+  def perform(date)
+    ldus = LocalDivisionalUnit.all.where.not(email_address: nil)
+
+    ldus.each do |ldu|
+      offenders = ldu.teams.map { |team| team.case_information.where(com_name: nil).map(&:nomis_offender_id) }.flatten
+                  .map { |off_id| OffenderService.get_offender(off_id) }
+                  .select { |o| o.handover_start_date.present? && o.handover_start_date == date - 1.week }
+
+      offenders.each do |offender|
+        allocation = Allocation.where(prison: offender.prison_id).first
```

Comment:
This looks like it picks up the wrong allocation when there are multiple allocations in a prison

Reply:
Rookie mistake :/